src/modules/m_cloaking.cpp

```diff
@@ -327,8 +328,11 @@ class ModuleCloaking : public Module
 		suffix = tag->getString("suffix", ".IP");
 
 		std::string modestr = tag->getString("mode");
-		if (modestr == "half")
+		if (modestr == "half") 
+		{
 			mode = MODE_HALF_CLOAK;
+			domainparts = tag->getInt("domainparts", 3);
```

Comment:
Mhm currently thinking what's the better min. `0` or `1` :thinking:

In the case of `0` full cloaking would be the better choice.

And there is no max by RFC. So I'm no sure if provide one is really perfect. In general, a max of 5 or 10 should be fine.